transport-native-epoll/src/main/c/netty_epoll_native.c

```diff
@@ -169,6 +169,31 @@ static jint netty_epoll_native_epollCreate(JNIEnv* env, jclass clazz) {
     return efd;
 }
 
+static void netty_epoll_native_timerFdSetTime(JNIEnv* env, jclass clazz, jint timerFd, jint tvSec, jint tvNsec) {
+    struct itimerspec ts;
+    memset(&ts.it_interval, 0, sizeof(struct timespec));
```

Comment:
nit: do you even need to meet as you assign all fields anyway

Reply:
the `it_interval` is not touched below so it is zero'ed out (we don't want a repeating timer).

```c
           struct timespec {
               time_t tv_sec;                /* Seconds */
               long   tv_nsec;               /* Nanoseconds */
           };

           struct itimerspec {
               struct timespec it_interval;  /* Timer interval */
               struct timespec it_value;     /* Initial expiration */
           };
```

Reply:
doh! missed this... good point